AppCenterReactNativeShared/ios/Podfile

```diff
@@ -0,0 +1,11 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'AppCenterReactNativeShared' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
```

Comment:
Are we sure we want to use dynamic frameworks here? 

Does `use_frameworks!` matter here? If you comment out `use_frameworks!`, would it still work?

I don't see `use_framework!` in TestApp and DemoApp, so I am not sure if we should or shouldn't use it.

Reply:
It doesn't build without this line, here are the errors I got:

```
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: can't locate file for: -lPods-AppCenterReactNativeShared

error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: file: -lPods-AppCenterReactNativeShared is not an object file (not allowed in a library)
```